src/server/db/singleNodeConfiguration.js

```diff
@@ -44,3 +44,11 @@ fluid.defaults("sjrk.storyTelling.server.globalChangesDb", {
         dbName: "_global_changes"
     }
 });
+
+// includes the users database name
+fluid.defaults("sjrk.storyTelling.server.storyUsersDb", {
```

Comment:
What is this "users" database? It doesn't seem to be used anywhere.

Reply:
I believe it's couchDB's user database for accounts to itself.

Reply:
oh actually this one might be a mistake, might have been leftovers from trying things out early on.